kerastuner/tuners/ultraband/new_ultraband.py

```diff
@@ -0,0 +1,65 @@
+import queue
+
+from kerastuner.engine import Tuner
+from kerastuner.engine.oracle import Oracle
+
+
+class UltraBandTuner(Tuner):
+    def run_trial(self, hp, trial_id, *fit_args, **fit_kwargs):
```

Comment:
Could you go ahead and start the docstrings for the methods you're implementing?  It'd help in following allong with what you're planning with the pseudocode.